frontend/src/pages/Dapp/Overview/Index.js

```diff
@@ -0,0 +1,38 @@
+import logo from "image/uniswap-logo.png";
+/* import linkIcon from "image/link.svg";
+import discord from "image/discord.svg";
+import twitter from "image/twitter.svg"; */
+import {NavLink} from "react-router-dom";
+
+const ProtocolCard = ({to, logo, alt, name}) => {
+  const url = `/dapp/overview/uniswap/${to}`;
+
+  return (
+    <NavLink to={url}>
+      <div className="rounded-lg bg-white p-5 shadow-md">
+        <div className="flex flex-col items-center">
+          <img className="bg-gray-100 rounded-full w-20 h-20 m-3" src={logo} alt={alt} />
+          <p className="text-3xl font-bold">{name}</p>
+        </div>
+      </div>
+    </NavLink>
+  );
+};
+
+const Index = () => {
+  return (
+    <main className="grid grid-cols-4 gap-6">
+      {[...Array(5)].map((_, i) => (
+        <ProtocolCard
+          to="uniswap"
```

Comment:
would be great to make it more composable:

- create state variable with 'useState' hook for all those infos
- create a function that will query data from ceramic, process it and push it in those state variable.

Reply:
Upgraded!